### DISCUSSION TITLE
Nextcloud AIO installation stuck on "starting container"

### DISCUSSION THREAD

**Discussion by HuengchI** [nextcloud/all-in-one#2312] [Questions] [answered]
# Problem
Hello, I am experiencing a common issue with my fresh installation of Nextcloud AIO. The container is getting stuck at the starting phase and not proceeding any further.

![image](https://user-images.githubusercontent.com/37769009/230760463-10e35739-d274-4401-be36-da8e55ed6d99.png)

# Logs
### 1.`docker logs nextcloud-aio-apache`
```
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
Waiting for Nextcloud to start...
```
### 2.`docker logs nextcloud-aio-nextcloud`
```
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n /dev/dri ']'
++ find /dev/dri -maxdepth 1 -mindepth 1 -name renderD128
+ '[' -n '' ']'
+ set +x
Installing imagemagick via apk...
```

**Comment by szaimen** [comment, marked as answer]
From the provided logs:
```
Initializing nextcloud 25.0.5.1 ...
Initializing finished
New Nextcloud instance.
Installing with PostgreSQL database
Starting Nextcloud installation...
Username is invalid because files already exist for this user
Installation of Nextcloud failed!

[...]

The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
```

**Reply by HuengchI** [reply]
Thank you very much for your response.

I followed the instructions in the documentation and performed a clean installation, but I'm still stuck at the `Installing imagemagick via apk...` step. As far as I understand, this step involves using Alpine's package manager to download some software, and I may have encountered some network issues. Is there any way to modify the Alpine `apk` software source? For example, by using a mirrored software source or a proxy.

**Reply by szaimen** [reply]
> I may have encountered some network issues

I would recommend to fix that instead by fixing your network infrastructure. There is no way to modify the Alpine apk software source.

**Comment by pepper-lees** [comment]
I'm having the same issue. Here's the nextcloud-aio-nextcloud log:

```
+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Configuring Redis as session handler...
Setting php max children...
The initial Nextcloud installation failed.
Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
```
which repeats ad infinitum.

I'm using WSL and my data directory is a mount to `E:\nextcloud-data`. This is my setup command:

```
docker run ^
--sig-proxy=false ^
--name nextcloud-aio-mastercontainer ^
--restart always ^
--publish 8080:8080 ^
--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config ^
--volume //var/run/docker.sock:/var/run/docker.sock:ro ^
-v E:\\nextcloud-data\\config\\config.php:/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/config.php ^
-e NEXTCLOUD_DATADIR="/run/desktop/mnt/host/e/nextcloud-data" ^
-e SKIP_DOMAIN_VALIDATION=true ^
-e SKIP_DATA_DIRECTORY_PERMISSION_CHECK=yes ^
nextcloud/all-in-one:latest
```
I've retried it many, many times. I don't know how to fix it. Please help if you have the chance @szaimen

**Reply by pepper-lees** [reply]
UPDATE: I tried using `nextcloud/all-in-one:beta` and the issue went away. (I did not check the checkbox that asked me if I wanted to update from version 25 to 26 or whatever it was)
